docs/reference/mapping/fields/synthetic-source.asciidoc

```diff
@@ -178,4 +157,38 @@ that ordering.
 
 [[synthetic-source-modifications-ranges]]
 ====== Representation of ranges
-Range field vales (e.g. `long_range`) are always represented as inclusive on both sides with bounds adjusted accordingly.  See <<range-synthetic-source-inclusive, examples>>.
+Range field vales (e.g. `long_range`) are always represented as inclusive on both sides with bounds adjusted accordingly. See <<range-synthetic-source-inclusive, examples>>.
+
+[[synthetic-source-precision-loss-for-point-types]]
+====== Reduced precision of `geo_point` values
+Values of `geo_point` fields are represented in synthetic `_source` with reduced precision. See <<geo-point-synthetic-source, examples>>.
+
+
+[[synthetic-source-fields-native-list]]
+===== Field types that support synthetic source with no storage overhead
```

Comment:
I'm on the fence on whether to report the fields that have additional indexing overhead when `ignore_malformed` or `ignore_above` are set. Wdyt?

Reply:
I think it's fair, i imagine normally you expect malformed values to be a small portion of your data.

Reply:
Well, the copying parser always copies the current value preemptively, iiuc?

Reply:
I see what you mean. I don't know if it should be in this section but we can mention it somewhere. Let me follow up.

Reply:
Let's wait with mentioning the copy parser overhead until we have a better understanding how much it actually is? I suspect it is minimal.